### DISCUSSION TITLE
Why doesn't ignores: ["dist/*.js"] work?

### DISCUSSION THREAD

**Discussion by Jakobud** [eslint/eslint#18646] [Q&A]
Here is my config

```
import js from "@eslint/js";

export default [
  js.configs.recommended,
  {
    files: ["src/**/*.ts"],
    ignores: ["dist/*.js"],
    languageOptions: {
      ecmaVersion: 2022,
      sourceType: "module",
    },
  },
];
```

Why isn't my `ignores` working? I have also tried `["dist/**/*.js"]`.

**Comment by alexnsorensen** [comment]
I was having the same issue, this [https://github.com/eslint/eslint/discussions/18304](https://github.com/eslint/eslint/discussions/18304) solved my issue
